source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/action/ActionElementRenderer.java

```diff
@@ -98,13 +156,62 @@ public void onClick(View v)
         private ViewGroup m_hiddenCardsLayout;
     }
 
+    private class ActionElementRendererImageLoaderAsync extends ImageLoaderAsync
+    {
 
+        protected ActionElementRendererImageLoaderAsync(RenderedAdaptiveCard renderedCard, View containerView, boolean iconIsAboveTitle)
```

Comment:
>iconIsAboveTitle [](start = 119, length = 16)

Just pass the enum instead of the bool? That way if we add a new position, this function signature doesn't need to change.